h1st/model/ensemble/stack_ensemble_modeler.py

```diff
@@ -55,17 +42,31 @@ def build_model(self, data: Dict[str, Any] = None) -> StackEnsemble:
                 'y_test': ...
             }
         """
-        if not data:
+        if data is None:
             data = self.load_data()
 
-        ensembler = self.train_base_model(data)
+        if data is not None:
+            X_train, y_train = (data['X_train'], data['y_train'])
+            X_train = self.model_class.preprocess(self._sub_models,
```

Comment:
@nhanitvn Do you intentionally only use sub models' output as an input for ensembler ? Could we concatenate original input X_train and sub models' output to make the input for ensembler ?

Reply:
I see. Should we make that behavior an option?

Reply:
Updated the preprocess() method to perform the concatenation.